tests/EvmAcceptanceTests/test/ERC20isZRC2.ts

```diff
@@ -34,18 +34,23 @@ describe("ERC20 Is ZRC2", function () {
       this.skip();
     }
 
-    contractOwner = await parallelizer.takeSigner();
-
-    zrc2_contract = await parallelizer.deployScillaContract(
-      "FungibleToken",
-      await contractOwner.getAddress(),
-      "ERC20isZRC2 Token",
-      "SDT",
-      2,
-      1_000
+    const signers = await ethers.getSigners();
+
+    if(signers.length < 3) {
+      this.skip();
```

Comment:
Why not `expect` this value? To fail if conditions are not satisfied, instead of ignoring it silently.